### PR TITLE
Extend smoke tests set

### PR DESCRIPTION
**Pull request by mmichal10**
Signed-off-by: Michal Mielewczyk <michal.mielewczyk@intel.com>

### CODE REVIEW

File: test/smoke_test/recovery/02 [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#
+# Copyright(c) 2012-2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+# DESCRIPTION Recovery test
```
Comment (micrakow) [inline test/smoke_test/recovery/02]: Some more description here would be nice to distinguish recovery tests

File: test/smoke_test/write_back/01 [hunk 2]
```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
+# Copyright(c) 2012-2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+# The line below specified that line under it should be used as the test's short description when launching test via run_tests script.
+# The text should not be longer than 80 chars - if it is, the script will strip addititonal characters
+# DESCRIPTION Copy file greater then cache
```
Comment (micrakow) [inline test/smoke_test/write_back/01]: than*

File: test/smoke_test/write_back/02 [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+. $TESTS_DIR/cas_lib
+start_test $*
+
+CACHE_DEVICE_SIZE=1G
+CORE_DEVICE_SIZE=2G
+TEST_DEVICE=${DEVICE_NAME}1-1
+TEST_FILE=/tmp/cas.testfile
+TEST_BS=32k
+TEST_COUNT=16
+#32768
```
Comment (micrakow) [inline test/smoke_test/write_back/02]: what no is that?

File: test/smoke_test/cache_suspend/01 [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+        L_FILE=${TEST_JOBS_FILE}${L_JOB_ID}-${I}
+        rm -f $L_FILE
+    done
+
+    echo $L_RESULT > $L_FILE_RESULT
+}
+
+#param1 - Flushing {YES - Perform Flushing during suspending cache, NO}
+cache_suspend() {
+    local L_CACHE_MODES=("wt" "wb" "wa" "pt" "wa")
```
[the remark this answers is not in the document]
Comment (arutk) [reply test/smoke_test/cache_suspend/01]: yes, please add WO :)

**Comment by KamilLepek** [comment]
retest this please

**Pull request merged**